libcontainer/factory_linux.go

```diff
@@ -34,12 +34,39 @@ var (
 // InitArgs returns an options func to configure a LinuxFactory with the
 // provided init binary path and arguments.
 func InitArgs(args ...string) func(*LinuxFactory) error {
-	return func(l *LinuxFactory) error {
+	return func(l *LinuxFactory) (err error) {
+		if len(l.InitArgs) == 0 {
```

Comment:
`len(args)`? Also I'm not sure if this check is necessary because there are cases where you the container command won't be executed (where you just do `runc create` and then `runc exec`).